src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -74,8 +75,15 @@
    private ProjectName named;
 
    @Inject
+   @WithAttributes(label = "Select Project", required = true)
+   private UISelectOne<String> projectNames;
```

Comment:
Instead of having a new input, it's better to call setCompleter in the existing ProjectName input